setup.cfg

```diff
@@ -52,7 +52,6 @@ install_requires =
     srsly>=2.4.1,<3.0.0
     catalogue>=2.0.6,<2.1.0
     typer>=0.3.0,<0.5.0
```

Comment:
```suggestion
    typer>=0.4.1,<0.5.0
```

Reply:
We want to try to keep the version ranges as wide as possible because people use spacy in combination with so many other tools and libraries. You can't specify particular pairs of versions of click/typer, so it's difficult to make this as specific as it should be.

Since typer v0.4.0 still works with many other versions of click, I'm not sure we want to be too restrictive here. We could consider specifying `typer>=0.3.0,!=0.4.0,<0.5.0`, but I'm not sure this is that helpful in the end.

The only places where we've forbidden intermediate versions in the range are due to security vulnerabilities, or where the dependency caused problems across the board (most recently: pytest v7.1.0).